### PR TITLE
Add more classes for check_class

### PR DESCRIPTION
**Pull request by nischalshrestha**
This PR adds a few more classes to check for when grading tables that have classes beyond the expected ones for `tibble`.

### CODE REVIEW

File: R/check_class.R [hunk 1]
```diff
@@ -179,6 +179,14 @@ hinted_class_message_list <- function() {
       exp_class = "grouped_df",
       message   = "Your table is a rowwise data frame, but I was expecting it to be grouped. Maybe you need to use `group_by()`?"
     ),
+    list(
+      exp_class = "py_grouped_df",
+      message   = "I was only expecting 1 value for each grouping in the table, but you have multiple values per grouping. Maybe you are missing a .groupby() call?"
+    ),
+    list(
+      obj_class = "py_grouped_df",
+      message   = "Your table rows are grouped, and not a numbered sequence. You can tell by the extra spacing around the column names. You can fix this with .reset_index()"
```
Comment (chendaniely) [inline R/check_class.R:188]: do we have a similar phrase in R when a tbl or dataframe has row names and we don't want the student to have rowname? that's essentially the same problem here.

Maybe rephrase the first sentence to:

> Your table rows (i.e index) are not a numbered sequence.

because even if the data frame is not grouped, you can still label the index and have the extra spacing.

Comment (chendaniely) [reply R/check_class.R:188]: Having said that, if this is only in the context for a grouped data frame, I think the current statement is fine?

Maybe still put in (i.e. index) in the phrase, since that is a term that is used a lot in pandas.

Comment (nischalshrestha) [reply R/check_class.R:188]: So from our chat yesterday, I don't think we hav made a feedback message regarding `rowname` in grading R data.frame yet.

I can do that rephrase of first sentence to:

> Your table rows (i.e Index) are not a numbered sequence.

Since `Index` is the actual name or do you think lowercase is fine?

Comment (nischalshrestha) [reply R/check_class.R]: @chendaniely, final rephrasing of this case:

> Your table row labels (i.e. index) are not a numbered sequence. You can tell by the extra spacing around the column names. You can fix this with .reset_index()

Let me know what you think.

Comment (chendaniely) [reply R/check_class.R]: Looks good?

**Review by rossellhayes — APPROVED** [review]
LGTM!
